sui-execution/v1/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -738,7 +738,7 @@ mod checked {
                     && !external_transfers.contains(&id)
                 {
                     return Err(ExecutionError::new(
-                        ExecutionErrorKind::SharedObjectWrapped,
+                        ExecutionErrorKind::SharedObjectOperationNotAllowed,
```

Comment:
I always forget (an am nervous about) this: Is Execution Error Kind part of the effects (in which case we shouldn't change this?)